resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,1981 @@
+                    updated: 2020-02-04T02:02:02
+                    movementReferenceNumber: MRN
+                    _links:
+                      self:
+                        href: /customs/transits/movements/arrivals/2
+                      messages:
+                        href: /customs/transits/movements/arrivals/2/messages
+                  retrievedArrivals: 2
+                  totalArrivals: 2
+        '406':
+          description: 'Not Acceptable'
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/errorResponse'
+                - example:
+                    code: ACCEPT_HEADER_INVALID
+              examples:
+                badRequest:
+                  description: The accept header is missing or invalid.
+                  value:
+                    code: ACCEPT_HEADER_INVALID
+      deprecated: false
+      security:
+        - userRestricted:
+            - common-transit-convention-traders
+
+  /customs/transits/movements/arrivals/{arrivalId}:
+    put:
+      tags:
+      - common-transit-convention-traders
+      summary: Resubmit an arrival notification message
+      description: >
+        Resend a notification to the Customs Office of Destination that a transit movement has arrived at its final destination.
+        
+        This request is also called an ‘Arrival Notification’ E_ARR_NOT (IE007) message.
+        
+        A successful response will contain a URI and an arrival ID for the notification, which you can use to retrieve the message status.
+        
+        **Note:** If an Economic Operators Registration and Identification (EORI) number has not been assigned to you, an HTTP 403 Forbidden error is returned.
+      operationId: ResubmitAnArrivalNotificationMessage
+      parameters:
+        - $ref: '#/components/parameters/contentTypeHeader'
+        - in: path
+          description: The arrival ID specifying the arrival to return.
+          name: arrivalId
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: '1'
+      responses:
+        '202':
+          description: 'Accepted'
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/CustomsTransitsMovementsArrivalsResponse2'
+                - description: JSON payload
+                  example:
+                    _links:
+                      self:
+                        href: /customs/transits/movements/arrivals/1
+                    arrivalId: '1'
+                    messageType: IE007
+                    body: <CC007A>...</CC007A>
+              example:
+                _links:
+                  self:
+                    href: /customs/transits/movements/arrivals/1
+                arrivalId: '1'
+                messageType: IE007
+                body: <CC007A>...</CC007A>

```

Comment:
Again, the Redocly linter flagged the absence of a `4XX` response for this operation.